### PR TITLE
Change ownership of test-evidence library

### PR DESCRIPTION
**Pull request by skisel-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -87,3 +87,4 @@ NOTICES     @garyverhaegen-da @dasormeter
 /libs-scala/nonempty-cats/src/    @S11001001
 /libs-scala/scala-utils/src/    @S11001001
 /libs-scala/scalatest-utils/src/    @S11001001
+/libs-scala/test-evidence/ @digital-asset/kv-participant @soren-da @matthiasS-da
```
Comment (stefanobaghino-da) [inline CODEOWNERS:90]: Looks like there's an error here. Does @soren-da exist? Does he have write access?

Comment (stefanobaghino-da) [reply CODEOWNERS:90]: Looks like @soren-da exists but doesn't have write access to the repo.

Comment (skisel-da) [reply CODEOWNERS:90]: I have already asked him, he will request an access and then this PR can be merged.

Comment (skisel-da) [reply CODEOWNERS:90]: he should be added now

**Review by stefanobaghino-da — APPROVED** [review]

**Pull request merged**